tools/db_bench_tool.cc

```diff
@@ -1229,6 +1229,9 @@ DEFINE_bool(
 DEFINE_bool(allow_concurrent_memtable_write, true,
             "Allow multi-writers to update mem tables in parallel.");
 
+DEFINE_bool(experimental_allow_mempurge, false,
```

Comment:
Where is this flag used in db_bench? I don't see any reference to it.

Reply:
My bad, I never pushed the last commit - sorry about that